### DISCUSSION TITLE
With prisma client, Deploy failed in the cloudflare dashboard but success in the cli.

### DISCUSSION THREAD

**Discussion by iflamed** [prisma/prisma#24094] [Q&A] [answered]
### Bug description

I use prisma orm in cloudflare worker environment, but deploy failed in the cloudflare dashboard and deploy success in the cli.
According the tech support from cloudflare, this worker script size is too large.
```
deploy_worker_failed:Your Functions script is over the 1 MiB size limit (workers.api.error.script_too_large)
```

### How to reproduce

Will you try this branch [https://github.com/iflamed/hono-react-ssr-shadcn-ui/tree/prisma-debug](https://github.com/iflamed/hono-react-ssr-shadcn-ui/tree/prisma-debug), Follow these command below will deploy success  in the cli, but failed on the cloudflare dashboard.
```
npm run install
npx prisma generate
npm run deploy
```

### Expected behavior

Deploy success in cloudflare dashboard.

### Prisma information

*schema.prisma*
```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["driverAdapters"]
}

datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
}
```

```typescript
const adapter = new PrismaD1(c.env.DB)
const prisma = new PrismaClient({ adapter }) // this line will result deploy failed.
```

### Environment & setup

- OS: Cloudflare Pages
- Database: Cloudflare D1
- Node.js version: Cloudflare worker

### Prisma Version

```
"@prisma/adapter-d1": "^5.13.0",
"@prisma/client": "^5.13.0",
```

**Comment by janpio** [comment]
The best way to get around this problem is to upgrade to a paid Cloudflare Workers account, that one has a higher limit which will not be hit so deployment should work.

Unfortunately Prisma is pretty big, so if your app itself also takes up some space it is pretty easy to hit that limit.

**Reply by iflamed** [reply]
Can we write this inside the document to help more people like me?

**Reply by iflamed** [reply, marked as answer]
Finally I switched to drizzle-orm. I was using Prisma, which is too big for cloudflare sites. For free users of cloudflare, there is a 1MB size limit for each worker script.
